TOC.md

```diff
@@ -9,8 +9,7 @@
   + [TiDB Introduction](/overview.md)
   + [What's New in TiDB 4.0](/whats-new-in-tidb-4.0.md)
   + [Basic Features](/basic-features.md)
-  + Compatibility
-    + [MySQL Compatibility](/mysql-compatibility.md)
+  + [MySQL Compatibility](/mysql-compatibility.md)
```

Comment:
In the future，maybe have other DB compatibility，so，i think  just leave it as it is

Reply:
If that happens in the future, we update the TOC and change it back :-)